Makefile

```
@@ -24,6 +24,14 @@ export TF_IN_AUTOMATION="true"
 export TF_VAR_MATURITY=${MATURITY}
 export TF_VAR_DEPLOY_NAME=${DEPLOY_NAME}
 PYTHON_VER ?= python3
+
+
+REPO_DIR := /CIRRUS-DAAC
+CIRRUS_DAAC_BRANCH := $(shell git -C $(REPO_DIR) rev-parse --abbrev-ref HEAD)
+CIRRUS_CORE_BRANCH := $(shell git rev-parse --abbrev-ref HEAD)
+export CIRRUS_CORE_VERSION=${CIRRUS_CORE_BRANCH}
+export CIRRUS_DAAC_VERSION=${CIRRUS_DAAC_BRANCH}
```

I believe these statements can be combined. https://www.gnu.org/software/make/manual/html_node/Variables_002fRecursion.html
```suggestion
export CIRRUS_DAAC_VERSION := $(shell git -C $(REPO_DIR) rev-parse --abbrev-ref HEAD)
export CIRRUS_CORE_VERSION := $(shell git rev-parse --abbrev-ref HEAD)
```